db/migrate/20150306172123_add_columns_to_dinners.rb

```diff
@@ -0,0 +1,9 @@
+class AddColumnsToDinners < ActiveRecord::Migration
+  def change
+    add_column :dinners, :dinner_name, :string
+    add_column :dinners, :date, :date
+    add_column :dinners, :time, :time
```

Comment:
Are you sure `:time` is the right time?  Why not store a single field called `starts_at` or whatever?